security-release-process-documentation/security-release-process.md

```diff
@@ -286,3 +321,357 @@ Yes.
+
+This includes all write access violations and exploitable reads.
+
+##### Information disclosure (targeted)
+
+Cases where the attacker can locate and read information from anywhere on the 
+system, including system information that was not intended or designed to be 
+exposed.
+
+- Remote Anonymous User
+    - Unauthorized access. Examples:
+        - Personally identifiable information (PII) disclosure
+        - Disclosure of PII (email addresses, cloud provider credentials)
+        - Attacker can collect PII without user consent or in a covert fashion
+
+#### Important
+
+Non-default critical scenarios or cases where mitigations exist that can help 
+prevent critical scenarios.
+
+##### Denial of service
+
+Must be "easy to exploit" by sending a small amount of data or be otherwise 
+quickly induced.
+
+- Remote Anonymous User 
+    - Persistent DoS. Examples:
+        - Sending a single malicious TCP packet results in service failure
+        - Sending a small number of packets that causes a service failure
+    - Temporary DoS with amplification. Examples:
+        - Sending a small number of packets that causes the system to be 
+          unusable for a period of time
+        - Server being down for a minute or longer
+        - A single remote client consuming all available resources 
+          (sessions, memory) on a server by establishing sessions and keeping them open
+
+- Authenticated User
+    - Persistent DoS against a high value asset. Example:
+        - Sending a small number of packets that causes a service failure for 
+          a high value asset in server roles 
+          (API server, etcd, controllers, etc), such as when a 
+          authenticated user can perform a DoS on etcd
+
+##### Elevation of privilege
+
+The ability to either execute arbitrary code or obtain more privilege than 
+authorized. 
+
+- Authenticated User
+    - Unauthorized access. Examples:
+        - arbitrary writing to the file system or server components like etcd,
+          where the user should not have the ability to write
+        - reading or writing to API objects where the user should not be able
+          to
+        - getting data from the server or server components, where the user
+          should not have the ability to read
+    - Execution of arbitrary code (remote code execution in the server)
+
+This includes all write access violations and exploitable reads.
+
+##### Spoofing
+
+An entity (computer, server, user, process) is able to masquerade as a 
+specific entity (user or computer) of his/her choice.
+
+Examples:
+- Service masquerades as the API server for the cluster and effectively
+  man-in-the-middle's the real API server.
+- API server uses client certificate authentication (SSL) improperly to allow 
+  an attacker to be identified as any user of his/her choice
+- RBAC bug that allows a malicious remote user to be seen as a different user 
+  of his or her choice
```

Comment:
These examples seem like they either fall under escalation of privilege or information disclosure... Maybe this section should be about auditing? I.e. the ability to mask an action as coming from someone else (or hide it completely)?